### PR TITLE
Update devise.en.yml

### PR DESCRIPTION
**Pull request by Dan13L3**
@baarkerlounger Reviewed these all in one go - let me know if anything looks odd

### CODE REVIEW

File: config/locales/devise.en.yml [hunk 1]
```diff
@@ -24,43 +24,43 @@ en:
...
   errors:
     messages:
-      already_confirmed: "was already confirmed, please try signing in"
-      confirmation_period_expired: "needs to be confirmed within %{period}, please request a new one"
-      expired: "has expired, please request a new one"
-      not_found: "not found"
-      not_locked: "was not locked"
+      already_confirmed: "Your account has already been confirmed. Sign in"
+      confirmation_period_expired: "Your account needs to be confirmed within %{period}. Request a new account"
+      expired: "Your account has has expired. Request a new account"
```
Comment (baarkerlounger) [inline config/locales/devise.en.yml:61]: @Dan13L3 this actually refers to reset password token not account, and doesn't need to be added into the message it's injected in the code. 

Same goes for the others in this section that do refer to account.

Comment (Dan13L3) [reply config/locales/devise.en.yml:61]: @baarkerlounger Ahh, apologies - is it simple enough to revert?

Comment (baarkerlounger) [reply config/locales/devise.en.yml:61]: Nothing merged yet so no need to revert anything. I think easiest is I'll try to put your intended changes into a new PR and you can double check?

Comment (Dan13L3) [reply config/locales/devise.en.yml:61]: @baarkerlounger sounds good - thanks

**Comment by baarkerlounger** [comment]
Superseded by https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/pull/461

**Pull request closed without merging**